src/main/daml/Daml/Finance/Settlement/Batch.daml

```diff
@@ -59,7 +61,7 @@ template Batch
             -- execute instruction and get used holding standards for the instrument
             (instructionCid, instruction) <- fetchByKey @Instruction instructionKey
             settledCid <- exercise (toInterfaceContractId @Instruction.I instructionCid)
-              Instruction.Execute with actors = actors `S.union` requestors
+              Instruction.Execute with actors = actors <> S.fromList (signatory this)
```

Comment:
very minot, but it's probably sligthly easier to read if written as `actors <> consenters <> singleton instructor`

Reply:
I changed it